package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "code-oss-dev",
   "version": "1.80.0",
-  "distro": "c523cf770e3f05498d5d51a3d7d1dd2d04375276",
+  "distro": "6a3d56295de301f79821b83e224ee769b0066981",
```

Comment:
this is pointing to branch in distro https://github.com/microsoft/vscode-distro/tree/ben/remote-checksums

Reply:
yeah I did that for testing only, will revert that part